pkg/nuclide-outline-view/lib/rpc-types.js

```diff
@@ -13,6 +13,7 @@ import type {TokenizedText} from '../../commons-node/tokenizedText-rpc-types';
 
 export type OutlineTree = {
   icon?: string, // from atom$Octicon (that type's not allowed over rpc so we use string)
+  kind?: string, // kind you can pass to the UI for theming, from OutlineTreeKind
```

Comment:
Ohh yeah.. unfortunately we don't have great docs about this :( `atom$Octicon` is not supported because that's a type from `flow-libs`, which the RPC framework doesn't know about. Rest assured that simple literal unions like this one should work fine.

There's an option in Nuclide settings called "Use local RPC" that is sufficient for testing this (just make sure Flow outlines still work, for example).